sw/applications/example_dma_sdk/main.c

```diff
@@ -12,28 +12,31 @@
 
 #include <stdint.h>
 #include <stdlib.h>
+#include <stdio.h>
 #include "dma_sdk.h"
+#include "dma.h"
```

Comment:
I understand there shstdio and dma can be removed, they are not necessary (checked)